### PR TITLE
Add Triple Splashdown fist as receiver to object dmg calc

### LINKED ISSUE #1656: Add triple splashdown hand damage receiver data
See also https://github.com/Sendouc/sendou.ink/issues/1396

### PR DESCRIPTION
**Pull request by ximk**
Closes #1656

### CODE REVIEW

**Comment by ximk** [comment]
Converting this PR into a draft as I've now learned TSD is affected by OS, and in the meantime I am going to ask dataminers for those multipliers.

**Comment by ximk** [comment]
Damage multiplier for OS has been added, should be good to go now.

File: app/features/object-damage-calculator/core/object-dmg.json
Comment (Sendouc) [inline app/features/object-damage-calculator/core/object-dmg.json:1]: I should document this better but this file should not be updated manually but instead generated by the `/create-object-dmg-json.ts`.

So we need to adjust the file above to generate the kind of `object-dmg.json` we want.

**Comment by ximk** [comment]
Working on formatting issues now, object-dmg.json is now generating correctly. Only thing now is that the formatting is off for some reason.

**Comment by Sendouc** [comment]
Oh yeah you can just run `biome:fix` script in the package.json to fix that.

So it's like content needs to come from the script but formatting comes from Biome.. :D yeah could be better

**Comment by ximk** [comment]
Fixed everything, should be good to go now!

image

**Comment by Sendouc** [comment]
Sorry took a long time to take a look but just one final question: where does the 100hp come from?

Otherwise looks good 💯

**Comment by ximk** [comment]
> Sorry took a long time to take a look but just one final question: where does the 100hp come from?

https://github.com/Leanny/splat3/blob/2e6337a95e146a89515ca380a0c850e786236c2a/data/parameter/810/weapon/WeaponSpPogo.game__GameParameterTable.json#L104

**Comment by Sendouc** [comment]
Ok cool! Thanks, good addition.

**Pull request merged**